src/palette/internal/palettecreator.cpp

```diff
@@ -216,7 +216,7 @@ PalettePtr PaletteCreator::newBeamPalette()
 PalettePtr PaletteCreator::newDynamicsPalette(bool defaultPalette)
 {
     PalettePtr sp = std::make_shared<Palette>(Palette::Type::Dynamic);
-    sp->setName(QT_TRANSLATE_NOOP("palette", "Dynamics"));
+    sp->setName(QT_TRANSLATE_NOOP("palette", "Dynamics & hairpins"));
```

Comment:
We don't want this change.

Reply:
Why not? That palette __does__ contain both

Reply:
We consider hairpins to be a form of dynamic.

Reply:
I don't

Reply:
That's a pity.

Reply:
Oh well. Doesn't translate nicely into German anyway.

Reply:
It is gone now.